src/loaders/svg/tvgXmlParser.cpp

```diff
@@ -61,10 +61,63 @@ string simpleXmlNodeTypeToString(SvgNodeType type)
         case SvgNodeType::Use: return "Use";
         case SvgNodeType::Video: return "Video";
         case SvgNodeType::ClipPath: return "ClipPath";
+        case SvgNodeType::Mask: return "Mask";
         default: return "Unknown";
     }
     return "Unknown";
 }
+
+bool isIgnoreUnsupportedLogElements(const char* tagName)
+{
+    const auto elementsNum = 1;
+    const char* const elements[] = { "title" };
+    const unsigned int elementsLength[] = { 5 };
```

Comment:
this looks unnecessary as we can do like this:
strncmp(tagName, elements[i], strlen(elements[i])  ...

Reply:
I changed the code.